bootstrap/src/main/config/mediaserver.xml

```diff
@@ -47,7 +47,10 @@
 	<resources>
 		<localConnection poolSize="100" />
 		<remoteConnection poolSize="50" />
-		<player poolSize="50" />
+		<player>
+			<poolSize>50</poolSize>
+			<cacheSize>100</cacheSize>
```

Comment:
Sure, update the project docs. Here is the page about configuration:
https://github.com/RestComm/mediaserver/blob/master/docs/sources-asciidoc/src/main/asciidoc/concept-chapter-Configuring_the_Media_Server.adoc